### PR TITLE
Fixed agent param

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-18877)

## Description
Added logic to set the agent parameter based on the platform we are running at such that it will match the data in Wildfire side.

### CODE REVIEW

File: Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py [hunk 1]
```diff
@@ -12,16 +11,27 @@ def __init__(self, base_url: str, verify: bool = True, proxy: bool = False, ok_c
...
+    @staticmethod
+    def get_agent():
+        """
+        Auto API expect the agent header to be 'xdr' when running from within XSIAM and 'xsoartim' when running from
+        within XSOAR (both on-prem and cloud).
+        """
+        platform = get_demisto_version().get('platform')
+        if platform == 'x2':
+            return 'xsoartim' if is_demisto_version_ge('8.0.0') else 'xdr'
+        return 'xsoartim'
```
Comment (JudahSchwartz) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py:26]:
```suggestion
        if platform == 'x2' and not is_demisto_version_ge('8.0.0') :
            return 'xdr'
        return 'xsoartim'
```

File: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_11.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks WildFire Reports
+Fixed an issue where the integration failed to send the right agent parameter when was used by XSIAM users.
```
Comment (JudahSchwartz) [inline Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_11.md:4]:
```suggestion
Fixed an issue where the integration failed to send the right agent parameter when used by XSIAM users.
```

**Review by JudahSchwartz — APPROVED** [review]
Noice. Minor comments

File: Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py:26]: Why do you think this is better?

Comment (JudahSchwartz) [reply Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py:26]: actually realized this is better, to flip it 
`

if   is_demisto_version_ge('8.0.0')  or platform == 'xsoar':
	            return 'xsoartim'
return 'xdr'

`

I think this is clearer than the nested if. there's 2 reasons why to return xsoar

**Pull request merged**